pom.xml

```diff
@@ -164,8 +164,24 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>subversion</artifactId>
-            <version>2.12.2</version>
+            <version>2.15.1</version>
             <optional>true</optional>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.tmatesoft.svnkit</groupId>
+                    <artifactId>svnkit</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.tmatesoft.svnkit</groupId>
+            <artifactId>svnkit</artifactId>
+            <version>1.10.4</version>
+        </dependency>
```

Comment:
this is not what you should have - you are now bundling all of svnkits dependencies in the plugin, at different versions from the subversion polugin.  yet they will all be unused (from this plugin) if subversion is intsalled.